protoutil/signeddata.go

```diff
@@ -80,3 +84,33 @@ func EnvelopeAsSignedData(env *common.Envelope) ([]*SignedData, error) {
 		Signature: env.Signature,
 	}}, nil
 }
+
+// LogMessageForSerializedIdentity returns a string with seriealized identity information,
+// or a string indicating why the serialized identity information cannot be returned.
+// Any errors are intentially returned in the return strings so that the function can be used in single-line log messages with minimal clutter.
+func LogMessageForSerializedIdentity(serializedIdentity []byte) string {
+	id := &msp.SerializedIdentity{}
+	err := proto.Unmarshal(serializedIdentity, id)
+	if err != nil {
+		return fmt.Sprintf("Could not unmarshal serialized identity: %s", err)
+	}
+	pemBlock, _ := pem.Decode(id.IdBytes)
```

Comment:
can we log at least the length of rest?

Reply:
Sorry, not understanding, log the length of what?

Reply:
of the rest, basically pem.Decode returns block, rest; but here we don't even assign a variable name to rest. However, for malformed PEM data, there might be trailing, undecoded data

Reply:
I haven't seen malformed PEM data in the last five years either, I think I'll just keep it simple.